client/verta/verta/client.py

```diff
@@ -112,10 +112,11 @@ class Client(object):
         Currently active Project.
     expt : :class:`Experiment` or None
         Currently active Experiment.
+    connect : connect to server (False for unit tests)
 
     """
     def __init__(self, host=None, port=None, email=None, dev_key=None,
-                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False):
+                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False, connect=True):
```

Comment:
This lets users know it's not generally for public use.
```suggestion
                 max_retries=5, ignore_conn_err=False, use_git=True, debug=False, _connect=True):
```